src/OnChain.ts

```diff
@@ -134,7 +134,7 @@ export const OnChainMixin = (superclass) =>
           if (payeeUser) {
             const onchainLoggerOnUs = onchainLogger.child({ onUs: true })
 
-            if (await this.user.limitHit({ on_us: true, amount })) {
+            if ((await this.user.remainingOnUsLimit()) < amount) {
```

Comment:
here `this.user.remainingOnUsLimit()` is a function, above it's a property? probably one of them is not right

Reply:
Nope, it is a method defined on the UserSchema. Are you referring to it being a field in the `userWallet.ts` file?